src/main/java/com/connectCo/domain/coupon/coupon.java

```diff
@@ -0,0 +1,46 @@
+package com.connectCo.domain.coupon;
+
+import com.connectCo.domain.BaseEntity.baseEntity;
+import com.connectCo.domain.enums.CouponType;
+import com.connectCo.domain.store.store;
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.util.Date;
+import java.util.UUID;
+
+@Getter
```

Comment:
```suggestion
@Getter
@Where(clause = "deleted_at is null")
```
우리가 엔티티를 하드 삭제가 아닌 소프트 삭제를 할 거니까 null이 아닌 경우 삭제 처리해주기 위해서 해당 where 어노테이션으로 엔티티 삭제 처리해주면 돼!!